db/migrate/20150610131245_change_start_and_end_time_fields_on_shift_to_datetime.rb

```diff
@@ -0,0 +1,15 @@
+class ChangeStartAndEndTimeFieldsOnShiftToDatetime < ActiveRecord::Migration
+  def up
+    remove_column :shifts, :starting_time, :time
+    remove_column :shifts, :ending_time, :time
+    add_column :shifts, :starting_time, :datetime
+    add_column :shifts, :ending_time, :datetime
+  end
+
+  def down
+    remove_column :shifts, :starting_tame, :datetime
+    remove_column :shifts, :ending_time, :datetime
+    add_column :shifts, :starting_time, :time
+    add_column :shifts, :ending_time, :time
+  end
+end
```

Comment:
I discussed this with @ansonK. The issue is that if you only store time the record gets retrieved by AR with a default db date of January 1st 2000. This caused comparing the time passed in from the "who is on duty" request to fail. Fixing the typo now :)